### PR TITLE
feat: build health declaration form

### PR DESCRIPTION
**Pull request by raymond-le-goldenowl**

### CODE REVIEW

File: src/app/components/Select/index.jsx [hunk 1]
```diff
@@ -3,14 +3,16 @@ import React, { useState } from 'react';
 
 import './styles.scss';
 
-const { AntOption = Option } = AntSelect;
+const { Option } = AntSelect;
```
Comment (jun-nguyen-goldenowl) [inline src/app/components/Select/index.jsx]: we should rename this component to `AntSelectOption`

File: src/app/components/Select/index.jsx [hunk 2]
```diff
@@ -35,13 +37,15 @@ export default function SelectOption({
...
 				}
-				defaultValue={defaultValue}
+				defaultValue={defaultValue || null}
+				disabled={disabled}
+				value={value}
 			>
 				{options &&
 					options.map(option => (
-						<AntOption key={option.key} value={option.value} style={{ fontSize: '1rem' }}>
+						<Option key={option.key} value={JSON.stringify(option)} style={{ fontSize: '1rem' }}>
```
Comment (jun-nguyen-goldenowl) [inline src/app/components/Select/index.jsx]: Why do we have to store value like this?

File: src/app/config/router/index.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		component: DeclarationForm,
+		layout: DeclarationFormWrapper,
+		content: 'Declaration Form'
+	},
+	default: {
+		id: nanoid(4),
+		path: '/',
+		component: () => <Navigate to={`/declaration-form`} />
+	}
+};
```
Comment (jun-nguyen-goldenowl) [inline src/app/config/router/index.js:26]: no idea why we don't use array instead

File: src/app/pages/DeclarationForm/styles.scss [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+					margin-right: 8px;
+				}
+
+				input[type='text'] {
+					width: 700%;
+				}
+			}
+		}
+	}
+}
```
Comment (jun-nguyen-goldenowl) [inline src/app/pages/DeclarationForm/styles.scss:363]: text indent is too large. Recommended text indent is 2 spaces

File: src/app/services/declarationTypes.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	},
+	{ id: nanoid(2), checked: false, value: 'Tiêm chủng vắc xin', text: 'Tiêm chủng vắc xin' },
+	{ id: nanoid(2), checked: false, value: 'Xét nghiệm Covid-19', text: 'Xét nghiệm Covid-19' },
+	{
+		id: nanoid(2),
+		checked: false,
+		value: 'Theo dõi sức khỏe tại nhà',
+		text: 'Theo dõi sức khỏe tại nhà'
+	}
+];
```
Comment (jun-nguyen-goldenowl) [inline src/app/services/declarationTypes.js:20]: fix text indent

File: src/index.js [hunk 6]
```diff
@@ -1,11 +1,21 @@
...
+	<React.StrictMode>
+		<Suspense fallback={<p>Loading ...</p>}>
+			<BrowserRouter history={history}>{renderRouteConfigs(routes)}</BrowserRouter>
+		</Suspense>
+	</React.StrictMode>,
+	document.getElementById('root')
+);
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
```
Comment (jun-nguyen-goldenowl) [inline src/index.js:21]: fix text indent

File: src/index.scss [hunk 7]
```diff
@@ -3,17 +3,20 @@
...
 body {
 	margin: 0;
-	font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
-		'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+	// font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu',
+	// 	'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+	font-family: 'Inter', 'Overpass', sans-serif;
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
 }
```
Comment (jun-nguyen-goldenowl) [inline src/index.scss:23]: text indent tooo

**Review by jun-nguyen-goldenowl — CHANGES_REQUESTED** [review]

File: src/app/components/Select/index.jsx (hunk 2 above)
Comment (raymond-le-goldenowl) [reply src/app/components/Select/index.jsx]: i set all data for option value i fetched before, i wanna get all them on select option, i don't wanna get an id and fetch data after select that it's waste times and i cant handle it easy way with data from owner api

File: src/app/config/router/index.js (hunk 3 above)
Comment (raymond-le-goldenowl) [reply src/app/config/router/index.js:26]: i don't know why code on repo show like that, i have config with eslint and prettier and also githook to but sometime it's still happening (just on repo, my code on local not showing like that -.-)

File: src/app/pages/DeclarationForm/styles.scss (hunk 4 above)
Comment (raymond-le-goldenowl) [reply src/app/pages/DeclarationForm/styles.scss:363]: i don't know why code on repo show like that, i have config with eslint and prettier and also githook to but sometime it's still happening (just on repo, my code on local not showing like that -.-)

File: src/app/config/router/index.js (hunk 3 above)
Comment (raymond-le-goldenowl) [reply src/app/config/router/index.js:26]: Just created like that to map with object-key-name,
Should i change it to an array?

**Review by jun-nguyen-goldenowl — APPROVED** [review]

**Pull request merged**